packages/kinvey-nativescript-sdk/lib/before-checkForChanges.js

```diff
@@ -0,0 +1,44 @@
+const path = require('path');
+const fs = require('fs');
+const url = require('url');
+const pkg = require('../package.json');
+
+module.exports = function (hookArgs) {
+  const appDirectoryPath = hookArgs && hookArgs.checkForChangesOpts && hookArgs.checkForChangesOpts.projectData && hookArgs.checkForChangesOpts.projectData.appDirectoryPath;
+
+  if (!appDirectoryPath) {
+    throw new Error('Unable to get path to app directory');
+  }
+
+  const pathToPackageJson = path.join(appDirectoryPath, 'package.json');
+  const packageJsonContent = JSON.parse(fs.readFileSync(pathToPackageJson));
+  const kinveyData = packageJsonContent.pluginsData && packageJsonContent.pluginsData[pkg.name];
+  const redirectUri = (kinveyData && kinveyData.config && kinveyData.config.redirectUri) || 'enterpriseauth://';
```

Comment:
Do we want to have a default here? In case there's no redirectUri in the package.json, the plugin's AndroidManifest.xml and Info.plist will register a scheme for `enterpriseauth`. In case you apply this for two applications, we will have the undefined behavior where multiple apps register the same scheme. 
As a safer solution, I suggest directly throwing an error in case there's no redirectUri in the package.json:
```
if (!redirectUri) {
    throw new Error(`Unable to find redirectUri info in ${pathToPackageJson}. Please provide correct redirectUri`);
}
```

Reply:
Fixed